index.js

```diff
@@ -0,0 +1,29 @@
+const { parse } = require('csv-parse')
+const fs = require('fs')
+const habitablePlanets = []
+
+function isHabitable (planet) { // checks if all the conditions to confirm that a planet is habitable are met
+  return planet.koi_disposition === 'CONFIRMED' &&

```

Comment:
lets give each of these a named variable. I'm not sure what koi_disposition or koi_insol etc means.

maybe something like 

``` const planetDiamter = planet.koi_prad```

something like the above is clearer

Reply:
koi_disposition and koi_insol are properties from the parsed stream of  'kepler_data.csv'.

Reply:
nice, lets name them something that anyone reading this can understand.

Googling it i found this: https://exoplanetarchive.ipac.caltech.edu/docs/API_kepcandidate_columns.html

koi_disposition means: 

Exoplanet Archive Disposition | The category of this KOI from the Exoplanet Archive. Current values are CANDIDATE, FALSE POSITIVE, NOT DISPOSITIONED or CONFIRMED. All KOIs marked as CONFIRMED are also listed in the Exoplanet Archive Confirmed Planet table. Designations of CANDIDATE, FALSE POSITIVE, and NOT DISPOSITIONED are taken from the Disposition Using Kepler Data.

So a good variable name could be:

`const doesExistInExoplantArchive = koi_disposition`

Feel free to think of names for the other ones